.github/workflow/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish to NPM
+
+on:
+  push:
+    tags:
```

Comment:
Do we want to publish when tags are pushed this or on [releases](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release)?

Reply:
Releases, but each release creates a tag. Happy to move this to a different event if you have one you prefer

Reply:
In fact yes, didn't follow your link but releases / published would be best